### ISSUE TITLE
Decimal.log("115.696") not close to Math.log(115.696)

### ISSUE THREAD

**Issue by jwboardman** [MikeMcl/decimal.js#19]
Decimal 4.0.3

log appears to be way off.

Math.log(115.696)
4.750966061432596

Decimal.log("115.696").toString()
"2.0633183441898021906"

I'd love to get rid of every Math.* call and use Decimal exclusively as I am working on a very high precision application. All the other functions I have used have worked great. Hopefully this can be fixed so I can use it too.

**Comment by josdejong** [comment]
How about 

```js
Decimal.ln("115.696").toString(); // "4.7509660614325960373"
```

**Comment by jwboardman** [comment]
Thank you. Good to know I was just missing something simple. Easy thing to
slip up on when both had a log() method, I should have been paying more
attention.

**Comment by MikeMcl** [comment]
Yes, it is confusing that `Math.log` is the natural log and `Decimal.log` is the logarithm to the base 10, with `Decimal.ln` instead being the natural log.

I have implemented most of the other `Math` methods now, such as `sin` etc. and hopefully I can get the new version published before the end of the year, after updating the docs etc.

**Comment by jwboardman** [comment]
I converted a VB app to Javascript, and Math.log with one argument in VB is
just like Javascript's, natural. I should have read your docs more closely,
but it's all good and working now.

You have created a great library. I was using BigNumber because i started
with it before Decimal came out. I've now switched completely to Decimal
and have been able to remove all Math calls.

Kudos for all your hard work!
